src/celeritas/em/model/WentzelModel.cc

```diff
@@ -49,17 +50,18 @@ WentzelModel::WentzelModel(ActionId id,
     host_data.ids.action = id;
     host_data.ids.electron = particles.find(pdg::electron());
     host_data.ids.positron = particles.find(pdg::positron());
-    host_data.ids.proton = particles.find(pdg::proton());
+    // TODO: Enable protons when supported
+    // host_data.ids.proton = particles.find(pdg::proton());
```

Comment:
You can revert this change too, since proton will just be "false" if it's not there.